pkg/webhook/service/service.go

```diff
@@ -10,13 +10,12 @@
 //   Red Hat, Inc. - initial API and implementation
 //
 
-package webhook
+package service
```

Comment:
With that dedicated package you solving cyclic import issue, right?

Reply:
Yeah